File: app/src/main/java/dev/ragnarok/fenrir/api/model/LocalServerSettings.java
package dev.ragnarok.fenrir.api.model;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class LocalServerSettings {
    @Nullable
    @SerializedName("url")
    public String url;

    @Nullable
    @SerializedName("password")
    public String password;

    @SerializedName("enabled")
    public boolean enabled;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/LocalServerSettings.kt
package dev.ragnarok.fenrir.api.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class LocalServerSettings {
    @SerialName("url")
    var url: String? = null

    @SerialName("password")
    var password: String? = null

    @SerialName("enabled")
    var enabled = false
}